bhohnco/Rancid-Tomatillos
# Feature cypress testing

## Is this a fix or a feature?
feature 

## What is the change?
* cypress has been added to the project
* cypress testing files have also been added
* the first 3 tests for the navbar are complete and passing 

## What does it fix?
N/A

## Where should the reviewer start?
The cypress integration file and navbar_spec.js file 

## How should this be tested?
Pull down the new files and see if cypress runs. Run the tests to see if they pass.
